distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -25,6 +25,11 @@
 #START DDF SETTINGS
 
 #
+# System branding
+#
+org.codice.ddf.system.branding=ddf
```

Comment:
Is there a way to use the branding defined https://github.com/codice/ddf/blob/17be0101150f311503a120f261ca709537ac4cee/pom.xml#L184, instead? Or, at least a way to make sure that this doesn't get out of sync with that property?

Reply:
Could maybe use maven property replacement/filtering to do that.